### ISSUE TITLE
AsciiDoctor aborts while typing wrong puml include

### ISSUE THREAD

**Issue by ascheman** [asciidoctor/asciidoctor-intellij-plugin#247]
While typing the following include I got the exception below.

````
[plantuml, svg, deployment]
----
include::deployment
````

I wanted to have the complete block to be

````
[plantuml, svg, deployment]
----
include::deployment.puml[]
----
````

It is clear that the file type (`svg`) should be the third parameter as in

````
[plantuml, deployment, svg]
----
include::deployment.puml[]
----
````

Nevertheless I would prefer an error message over an exception ;-)

```
Error rendering preview: org.jruby.exceptions.RuntimeError: (RuntimeError) asciidoctor: FAILED: <stdin>: Failed to load AsciiDoc document - Asciidoctor::Diagram::PlantUmlBlockProcessor does not support output format deployment

org.asciidoctor.internal.AsciidoctorCoreException: org.jruby.exceptions.RuntimeError: (RuntimeError) asciidoctor: FAILED: <stdin>: Failed to load AsciiDoc document - Asciidoctor::Diagram::PlantUmlBlockProcessor does not support output format deployment
	at org.asciidoctor.internal.JRubyAsciidoctor.convert(JRubyAsciidoctor.java:421)
	at org.asciidoctor.internal.JRubyAsciidoctor.convert(JRubyAsciidoctor.java:384)
	at org.asciidoctor.internal.JRubyAsciidoctor.render(JRubyAsciidoctor.java:210)
	at org.asciidoc.intellij.AsciiDoc.render(AsciiDoc.java:204)
	at org.asciidoc.intellij.editor.AsciiDocPreviewEditor$2.run(AsciiDocPreviewEditor.java:159)
	at org.asciidoc.intellij.editor.LazyApplicationPoolExecutor$1.run(LazyApplicationPoolExecutor.java:59)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:311)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.jruby.exceptions.RuntimeError: (RuntimeError) asciidoctor: FAILED: <stdin>: Failed to load AsciiDoc document - Asciidoctor::Diagram::PlantUmlBlockProcessor does not support output format deployment
	at RUBY.process(uri:classloader:/gems/asciidoctor-diagram-1.5.12/lib/asciidoctor-diagram/extensions.rb:108)
	at RUBY.process(<script>:14)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:129)
	at RUBY.build_block(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:1038)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.next_block(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:866)
	at RUBY.next_section(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:369)
	at RUBY.parse(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:98)
	at RUBY.parse(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/document.rb:570)
	at RUBY.load(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor.rb:1367)
	at RUBY.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor.rb:1479)
```

**Comment by ahus1** [comment]
Please add the version of the intellij plugin you are using to the description. 

You could give the latest pre-release on of this plugin a try (available via GitHub releases): this includes Asciidoctor 2 (instead of 1.5.8) that might (or might not) fix the issue.

**Comment by ascheman** [comment]
It's Plugin version 0.24.4

**Comment by ascheman** [comment]
Had similar problems with a current local build of the plugin (0.25.6?)

**Comment by ahus1** [comment]
0.25.6 is still work in progress. I've just pushed a change that in 0.25.6 this will no longer show as an IDE error. 

Still, with the current implementation in Asciidoctor Diagram the line number is not reported. I'll contact the upstream project.

**Comment by ahus1** [comment]
Issue with upstream project opened. Closing this issue for now.

**Comment by ahus1** [comment]
I named the wrong release in my previous comment. 

This change is available from release 0.25.7 and is available from the [Github Releases tab](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) as a download.
